plugin/src/main/java/io/snyk/eclipse/plugin/properties/PreferencesPage.java

```diff
@@ -30,22 +32,6 @@ public void init(IWorkbench workbench) {
 
   @Override
   protected void createFieldEditors() {
-    File lsFile = new LsRuntimeEnvironment(PREFERENCES).getLSFile();
-    String msg = "Snyk Language Server not found at " + lsFile.getAbsolutePath();
```

Comment:
The default path is never displayed anywhere anymore, which is a regression. It should probably be displayed in the Snyk LS path field. Clients have already needed and used that information a lot for troubleshooting and deleting LS from that location to trigger a fresh download.